experimental/dds/tree2/src/class-tree/schemaTypes.ts

```diff
@@ -109,13 +112,25 @@ export interface TreeNodeSchemaClass<
  * @beta
  */
 export interface TreeNodeSchemaCore<
-	out Name extends string = string,
-	out Kind extends NodeKind = NodeKind,
-	out Specification = unknown,
+	out Name extends string,
+	out Kind extends NodeKind,
+	out ImplicitlyConstructable extends boolean,
 > {
 	readonly identifier: Name;
 	readonly kind: Kind;
-	readonly info: Specification;
+	readonly info: unknown;
+
+	/**
+	 * If true, when constructing insertable content,
```

Comment:
Nit: this phrasing is a bit hard to parse. Might read a bit easier as something like "If true, the data that would have been passed to the node's constructor can be included instead of an unhydrated node when constructing insertable content."

Reply:
fixed, I think.